src/main/app/src/app/admin/groep-signaleringen/groep-signaleringen.component.ts

```diff
@@ -30,8 +30,9 @@ export class GroepSignaleringenComponent
   groepen: Observable<Group[]>;
   groepId: string;
   columns: string[] = ["subjecttype", "type", "dashboard", "mail"];
-  dataSource: MatTableDataSource<SignaleringSettings> =
-    new MatTableDataSource<SignaleringSettings>();
+  dataSource = new MatTableDataSource<
+    GeneratedType<"RestSignaleringInstellingen">
```

Comment:
is this the correct formatting? Looks weird to me 😉

Reply:
It is, spotless made it this way.

`dataSource = new MatTableDataSource<GeneratedType<"RestSignaleringInstellingen">>();` is just too long to be on the same line for formatting